uv.lock

```diff
@@ -2764,7 +2764,7 @@ requires-dist = [
     { name = "catkin-pkg", marker = "sys_platform == 'linux'", specifier = "==1.0.0" },
     { name = "click", specifier = ">=8.2" },
     { name = "craft-application", extras = ["remote"], specifier = ">=5.6.1,<6.0.0" },
-    { name = "craft-archives", specifier = "~=2.1" },
+    { name = "craft-archives", specifier = ">=2.1.1,<3.0.0" },
```

Comment:
the diff here does not update craft-archives on the lockfile, is it already pointing to `2.2.0`?

Reply:
Wow, craft-archives was bumped to 2.2.0 in https://github.com/canonical/snapcraft/pull/5699

I don't see anything in 2.2.0 that concerns Snapcraft, but I'd prefer not to bump craft libraries on a hotfix branch unless absolutely required.